pkg/components/components.go

```diff
@@ -465,34 +461,33 @@ func GetClusterPermissions() []rbacv1.PolicyRule {
 		},
 		roleWithAllPermissions("monitoring.coreos.com", stringListToSilce("servicemonitors", "prometheusrules")),
 		{
-			APIGroups: []string{
-				"operators.coreos.com",
-			},
+			APIGroups: stringListToSilce("operators.coreos.com"),
 			Resources: stringListToSilce("clusterserviceversions"),
 			Verbs:     stringListToSilce("get", "list", "watch"),
 		},
 		{
-			APIGroups: []string{"scheduling.k8s.io"},
+			APIGroups: stringListToSilce("scheduling.k8s.io"),
```

Comment:
Not specific to this PR,
but we have a typo in the function name.
Better to address this in a separate PR for visibility.